cmd/validate-krew-manifest/main.go

```diff
@@ -77,6 +78,16 @@ func validateManifestFile(path string) error {
 	}
 	glog.Infof("structural validation OK")
 
+	if maxLen := 80; !validateLineLength(p.Spec.Description, maxLen) {
+		return fmt.Errorf("line length in `description` exceeds %d characters", maxLen)
+	}
+	if maxLen := 70; !validateLineLength(p.Spec.ShortDescription, maxLen) {
```

Comment:
IMHO, `isValidLineLength` signals what it does better.

Reply:
I agree, the function name is o_O